src/main/java/florizz/core/FuzzyLogic.java

```diff
@@ -211,75 +222,82 @@ protected static String processCommand(String userInput) throws FlorizzException
      * @throws FlorizzException if there is an issue processing the input.
      */
     protected static String splitAndMergeInput(String userInput) throws FlorizzException {
```

Comment:
Throws unhandled exception for info and compare

Reply:
<img width="848" alt="image" src="https://github.com/AY2324S2-CS2113-T11-3/tp/assets/110618615/493f7f05-7366-45f1-af53-a9bdd210656a">

Fixed Exception Handling